ignite/handlers/state_param_scheduler.py

```diff
@@ -329,21 +329,43 @@ class StepStateScheduler(StateParamScheduler):
 
     Examples:
 
-        .. code-block:: python
+        .. testsetup::
 
-            ...
-            engine = Engine(train_step)
+            engine = get_default_trainer()
+
+        .. testcode::
+
+            from ignite.handlers.state_param_scheduler import StepStateScheduler
```

Comment:
```suggestion
```
Useless.

In the conf.py of the doc : 
https://github.com/pytorch/ignite/blob/73b9d1ff390d4c253cc9fb440b484f9a193c2c11/docs/source/conf.py#L341

In ignite.handler.__init__.py
https://github.com/pytorch/ignite/blob/73b9d1ff390d4c253cc9fb440b484f9a193c2c11/ignite/handlers/__init__.py#L21